Hudl.Mjolnir.Tests/Command/Attribute/NewCommandAttributeAndProxyTestsIgnoringTimeouts.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.Threading;
+using Hudl.Mjolnir.Command.Attribute;
+using Hudl.Mjolnir.Tests.Helper;
+using Xunit;
+
+namespace Hudl.Mjolnir.Tests.Command.Attribute
+{
+    public sealed class NewCommandAttributeAndProxyTestsIgnoringTimeouts : TestFixtureIgnoreTimeouts
```

Comment:
I'm thinking that relying on assumptions in a unit test for it to pass, is a little bit flawed. Also if one test forgets to tear down properly then finding where that is could be more tricky.  

But on the other hand if we actually changed the default values for config inside of mjolnir in the future, then I would hope a bunch of failing tests would be enough for someone to reconsider what they're doing. This is probably the biggest benefit so I'll go with the approach of just setting values where they are needed and assuming that the world is otherwise configured using defaults.